pvl/decoder.py

```diff
@@ -16,14 +16,38 @@ class ParseError(ValueError):
     colno: The column corresponding to pos
     """
     def __init__(self, msg, pos, lineno, colno):
-        errmsg = '%s: line %d column %d (char %d)' % (msg, lineno, colno, pos)
+        if None not in (pos, colno):
+            errmsg = '%s: line %d column %d (char %d)' % (
+                msg, lineno, colno, pos)
+        else:
+            errmsg = '%s: line %d' % (msg, lineno)
         super(ParseError, self).__init__(errmsg)
         self.msg = msg
         self.pos = pos
         self.lineno = lineno
         self.colno = colno
 
 
+class EmptyValue(str):
+
```

Comment:
This class might benefit from a docstring describing why it exists and how to use it (or how it's used).  For instance, I see these `EV(1)`, `EV(2)`, and `EV(3)`.   I don't get what the numbers are.